eus_cddlib/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+project(eus_cddlib)
+
+cmake_minimum_required(VERSION 2.4.6)
+
+find_package(catkin REQUIRED COMPONENTS rostest)
+
+catkin_package()
+
+add_definitions(-DGMPRATIONAL)
+add_library(eus_cddlib SHARED src/eus_cddlib.c)
+set_target_properties(eus_cddlib PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib)
+target_link_libraries(eus_cddlib cddgmp)
+install(DIRECTORY lib/
```

Comment:
I think we do not need this because we already have "install(TARGETS eus_cddlib"